nginx/requirements.txt

```diff
@@ -0,0 +1 @@
+# integration pip requirements
```

Comment:
I think the concern is perfectly valid, and I'm fine from that. But from the contibutor's perspective (maybe more relevant for `integrations-extra`) if the SDK philosophy picks up, I find it _very_ likely that we will end up some contributor adding it to the `requirements.txt` and maybe merging it without realizing (we'd quickly detect `requests` but maybe not some more obscure dependency). It should be easy to discern if a package is available in the core and disregard during the build process I believe. 

Also, although it's definitely not desireable to ship the same package twice, we should still ensure that any package versions listed in the `requirements.txt` don't clash with the `dd-agent`'s, as a matter of fact we might already be doing this in the `rake requirements` task.... but I have to double check. It should be relatively easy to add any missing logic to the task if necessary to enforce that.